fairing/builders/cluster/gcs_context.py

```diff
@@ -30,29 +29,9 @@ def upload_context(self, context_filename):
                     bucket_name=self.gcp_project,
                     blob_name='fairing_builds/' + context_hash,
                     file_to_upload=context_filename)
-        
-    def create_secret(self):
```

Comment:
@r2d4 Is it the case that based on our analysis of all paths a user may take (local/KF/hosted notebooks), there won't be a need to create secret?

Reply:
The (possible) secret generation was moved into a pod_spec_mutator - the GCS context preparer shouldn't be responsible for creating this secret.